dali/ft/filecopy.cpp

```diff
@@ -1513,6 +1517,37 @@ bool FileSprayer::usePushWholeOperation() const
     return targets.item(0).filename.isUrl();
 }
 
+bool FileSprayer::useAPICopy()
+{
+    if (isContainerized()) // Future: support in bare-metal?
+    {
+        bool needCalcCRC = calcCRC();
+        if (needCalcCRC && !sources.item(0).hasCRC)
+            return false;
+
+        StringBuffer targetClusterName, sourceClusterName;
+        distributedTarget->getClusterName(0, targetClusterName); // change this to support logical files in > 1 clusters
+        Owned<IStoragePlane> targetStoragePlane = getDataStoragePlane(targetClusterName.str(), false);
+        if (targetStoragePlane==nullptr)
+            return false;
+        distributedSource->getClusterName(0, sourceClusterName);
```

Comment:
picky style:  A few extra newlines make it easier to read.  E.g. after each separate test.  (sourceClusterName can also be declared later on, closer to where it is used.)